### PR TITLE
`pet-image-detection` added to data science section

### PR DESCRIPTION
**Pull request by isabelizimm**
created "Other Projects" tab under Data Science, added [https://github.com/aicoe-aiops/pet-image-detection](https://github.com/aicoe-aiops/pet-image-detection)

### CODE REVIEW

**Comment by oindrillac** [comment]
What would be the goal of including this in an "Other Projects" tab in the data science side panel and not in the main tab?
cc: @isabelizimm @MichaelClifford

File: config/data-science.yaml [hunk 1]
```diff
@@ -54,3 +54,7 @@
       href: /data-science/mailing-list-analysis/notebooks/02_analyses/contributor_analysis.ipynb
     - label: Contributor Analysis
       href: /data-science/mailing-list-analysis/notebooks/02_analyses/keyword_analysis.ipynb
+- label: Other Projects
```
Comment (MichaelClifford) [inline config/data-science.yaml]: I don't think we need to create a new "Other Projects" section. Can you add this to the "Data Science Projects" tab at the top?

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

**Comment by isabelizimm** [comment]
> What would be the goal of including this in an "Other Projects" tab in the data science side panel and not in the main tab?
> cc: @isabelizimm @MichaelClifford

My goal was to have a collecting point for all smaller projects that exist as just 1 notebook or so to compartmentalize. I updated it to be its own header.

**Comment by MichaelClifford** [comment]
@isabelizimm It's still unclear to me why this needs its own section? There is already a Data Science Projects tab that contains smaller projects. I think it would be appropriate to add your work there. : )

**Comment by isabelizimm** [comment]
That works for me! I'm okay with it wherever it fits best :D

**Comment by aakankshaduggal** [comment]
@isabelizimm I have added some changes to this file, please make sure to `git pull --rebase upstream master` before you push your updates :smile:

**Comment by isabelizimm** [comment]
/assign @martinpovolny
(does this need to be changed with Martin's move?)

**Review by oindrillac — COMMENTED** [review]
since @isabelizimm has 2 docs to add, having a separate section should be fine.
/lgtm

**Pull request merged**